returnn/torch/engine.py

```diff
@@ -3,15 +3,14 @@
 """
 
 from __future__ import annotations
-from typing import Optional, Any, Union, Callable, Dict
+from typing import Optional, Any, Union, Dict
 from contextlib import nullcontext
 
 import sys
 import gc
 import os
 import torch
 import time
-from torch.distributed import init_process_group
```

Comment:
I really wonder, I think you are right, I also don't see any usage of this now, but why did PyCharm not complain about this?

Reply:
Ah, but now, you see those errors:
```
  returnn/torch/engine.py:311: WARNING PyUnresolvedReferencesInspection: Cannot find reference 'distributed' in '__init__.py | __init__.py'
  returnn/torch/engine.py:311: WARNING PyUnresolvedReferencesInspection: Cannot find reference 'distributed' in '__init__.py | __init__.py'
  returnn/torch/engine.py:426: WARNING PyUnresolvedReferencesInspection: Cannot find reference 'distributed' in '__init__.py | __init__.py'
  returnn/torch/engine.py:491: WARNING PyUnresolvedReferencesInspection: Cannot find reference 'distributed' in '__init__.py | __init__.py'
```
So it means that `torch.distributed` is not available by default, you need to import it.

Reply:
But just doing this should be enough:
```python
import torch.distributed
```